apps/web/app/(use-page-wrapper)/settings/(settings-layout)/SettingsLayoutAppDirClient.tsx

```diff
@@ -27,6 +29,8 @@ import type { VerticalTabItemProps } from "@calcom/ui/components/navigation";
 import { VerticalTabItem } from "@calcom/ui/components/navigation";
 import { Skeleton } from "@calcom/ui/components/skeleton";
 
+type TeamFeatures = Record<number, Record<keyof AppFlags, boolean>> | null;
```

Comment:
```suggestion
type TeamFeatures = Record<number, Record<keyof AppFlags, boolean>>;
```

We usually do `teamFeatures?: TeamFeatures` anyways